integrationTests/relayers/ethToMultiversXWithChainSimulator_test.go

```diff
@@ -757,13 +820,13 @@ func issueAndWhitelistToken(
 
 	log.Info("multi-transfer set max bridge amount for token tx executed", "hash", hash, "status", txResult.Status)
 
-	return newUniversalToken
+	return newUniversalToken, newChainSpecificToken
```

Comment:
a little bit misleading that we return 2 string parameters here. Not having a better solution, thus.